docs/markdown/Java and Scala/kotlin.md

```diff
@@ -0,0 +1,223 @@
+---
+title: "Kotlin"
+slug: "kotlin"
+excerpt: "Kotlin support for Pants."
+hidden: false
+---
```

Comment:
I created this page for you in the Readme editor for 2.13 and 2.14

I think you also want to add this:

```
createdAt: "2022-07-26T02:02:59.962Z"
updatedAt: "2022-07-26T02:02:59.962Z"
```